### PR TITLE
de-1129 Add route to webhook types endpoint

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: src/terrain/routes/webhooks.clj [hunk 1]
```diff
@@ -10,4 +10,6 @@
     [config/app-routes-enabled]
 
     (PUT "/webhooks" [:as {:keys [body]}]
-      (service/success-response (apps-client/save-webhooks body)))))
+      (service/success-response (apps-client/save-webhooks body)))
+    (GET "/webhooks/types" [:as {:keys [params]}]
```
Comment (slr71) [inline src/terrain/routes/webhooks.clj]: This isn't required, but I normally add a blank line between route definitions.

Comment (ianmcorvidae) [inline src/terrain/routes/webhooks.clj]: I suspect you don't need anything inside the `[]` here, since you're not using it below.

Comment (slr71) [reply src/terrain/routes/webhooks.clj]: Ah, good catch.

**Comment by sri2k1us** [comment]
Thanks for the reviews. Pushed changes to address the feedback.

**Pull request merged**